Aquifer/Proxy.swift

```diff
@@ -82,6 +49,51 @@ public struct Proxy<UO, UI, DI, DO, FR> {
     }
 }
 
+/// A `Proxy` is so named because it can represent many different kinds of information flows.  There
```

Comment:
If I do that Quick Help won't show each specific flow, and I can't think of a way to merge the two... Maybe it's best to go back to the fake typealias stuff and produce new docs for the enums?